extra/systemd/codiusd-gui.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Codiusd GUI
+After=network.target nss-lookup.target codiusd
```

Comment:
The `nss-lookup.target` argument should be `codiusd-gui` instead of `codiusd`

Reply:
i dont think so? it just means it will start after networking and codiusd have been started, which in the order of things makes sense no?